docs/sources/developers/plugins/extend-the-grafana-ui-with-links.md

```diff
@@ -1,46 +1,57 @@
 ---
-title: Extend the Grafana UI with links
-description: Learn how to add links to the Grafana user interface from an App plugin
+title: Use extensions to add links to app plugins
+description: Learn how to add links to the Grafana user interface from an app plugin
 ---
 
-# Extend the Grafana UI with links
+# Use extensions to add links to app plugins
 
-Use the Plugin extensions API with your Grafana App plugins to add links to the Grafana UI. Doing so allows you to direct users to your plugins pages from various "extension points" within the Grafana application.
+You can use the Plugin extensions API with your Grafana app plugins to add links into the Grafana UI. This feature allows you to send users to your plugin's pages from other spots in the Grafana application.
 
-For a plugin to successfully register links it must:
+## Before you begin
 
-- Be an App plugin.
-- Be [preloaded]({{< relref "./metadata" >}}).
-- Be installed and enabled.
+Be sure your plugin meets the following requirements before proceeding:
+
+- It must be an app plugin.
+- It must be [preloaded with metadata]({{< relref "./metadata" >}}) for your plugin.
```

Comment:
I think we should probably change this line (it can only be set to be "preloaded", the metadata refers to the settings you can set in the `plugin.json` config).

Maybe something like this?
```suggestion
- It must be preloaded (by setting the [preload property]({{< relref "./metadata" >}}) to `true` in the `plugin.json`
```